src/Makefile.am

```diff
@@ -808,8 +808,7 @@ tests_testMem_LDFLAGS = $(LIBADD_DL)
 
 check_PROGRAMS += tests/testCharacterSet
```

Comment:
It feels like the no-header should be the default for unit test classes. Thus, I suggest adjusting `src/tests/testBoilerplate.*` accordingly. I do not insist on that adjustment (until there are at least some test classes that legitimately use header files), but adding a comment to the boilerplate class about the need for a header file would be nice.

Reply:
I agree.

Reply:
Adding a comment to testBoilerplate.h